src/main/resources/application.yml

```diff
@@ -84,14 +84,20 @@ notify:
     end-of-service-report-submitted: "f79cb53f-dc5c-42aa-947e-f599f9ad7942"
 
 interventions-ui:
+  # links added to email notifications
   locations:
```

Comment:
Not sure links are naturally categorised to "email links" and "delius links". There will be overlap

Can we use `probation-links` and `provider-links` as groups? E.g. `interventions-ui.probation-links.session-feedback` and `interventions-ui.provider-links.submit-action-plan`